configs/components/puppet-ca-bundle.rb

```diff
@@ -1,17 +1,39 @@
 component "puppet-ca-bundle" do |pkg, settings, platform|
   pkg.load_from_json("configs/components/puppet-ca-bundle.json")
 
-  pkg.build_requires "openssl"
+  pkg.build_requires "openssl-#{settings[:runtime_project]}"
+
+  java_available = true
```

Comment:
I copied this java check from the puppet-agent repo; Is this acceptable to have here, or should I separate this into two different `puppet-ca-bundle` components?

Reply:
This won't work for PDK, can we just set that variable to 'true' based on which runtime we're building, rather than splitting them apart?